packages/cli/templates/webcomponents/igc-ts/projects/side-nav/files/index.html

```diff
@@ -17,7 +17,7 @@
       padding: 25px;
     }
 
-    #outlet {
+    app-root {
```

Comment:
Also also, is that supposed to be part of the app component itself?

Reply:
No, we need to apply those styles to the <app-root> element in index.html, too.